app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,38 @@
+class Users::ConfirmationsController < Devise::ConfirmationsController
+  # GET /resource/confirmation/new
```

Comment:
If you aren't planning to override these methods soon you should remove the commented code.